testinput_tier_1/sensor_central_wavenumber_iasi.csv

```diff
@@ -0,0 +1,618 @@
+channel,sensorMetaData/sensorCentralWavenumber
+int,float

```

Comment:
if this going to be int float you need to put the chanel number in front of the frequency or you could just put float

Reply:
Not sure I follow. Isn't that how it currently is? where channel starts at 16 and wavenumber starts at 64875.0?

Reply:
oh, and I noticed this so late.. sorry @smaticka 
but a wee-niggle is that we may want to rename this `iasi_sensor_central_wavenumber` as then it will line up with the other `iasi` files?

Reply:
@PatNichols, `16,64875.0` is for channel number 16 with central wavenumber 64875.